pkg/apis/options/options.go

```diff
@@ -40,6 +42,9 @@ type Options struct {
 
 	// Not used in the legacy config, name not allowed to match an external key (upstreams)
 	// TODO(JoelSpeed): Rename when legacy config is removed
+
+	TenantLoader tenant.LoaderConfiguration `cfg:",internal"`
```

Comment:
Please don't add runtime information to structs that are intended for configuration

Reply:
this is the configuration for tenantLoader, not the runtime construct,

Reply:
resolved, remved the runtime bits away from the options package.